homeassistant/components/cpuspeed/strings.json

```diff
@@ -0,0 +1,14 @@
+{
+  "title": "CPU Speed",
```

Comment:
```suggestion
```

Not needed since its the same as the manifest

Reply:
Incorrect, this is an integration that is perfectly fit for offering translation on the title.